PolyEngine/Standalone/macOS/Src/CppInterface.cpp

```diff
@@ -0,0 +1,78 @@
+#include "CppInterface.hpp"
+
+#include <memory>
+
+#include <Core.hpp>
+#include <Engine.hpp>
+#include <GLRenderingDevice.hpp>
+#include <Game.hpp>
+
+using namespace Poly;
+
+static std::unique_ptr<Engine> engine;
+
+#include <sstream>
+
+IGame* LoadGame()
+{
+    // TODO dynamic dylib loading
```

Comment:
Turns out that macOS does actually conform to POSIX (hell, it's even certified) and the code for dynamic loading is line-for-line the same as on Linux (unless you want to load Apple Frameworks written in Objective-C, then it gets weird)